Automattic/jetpack
# tools: Lint tools and .github as 8.2+

## Proposed changes:

In #34207 we added Jetpack-Compat-82 to the .phpcs.dir.xml for `tools/` and `.github/` since those should generally always be being run with a version of PHP as specified in PHP_VERSION.

The same logic applies to the linting checks.

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack/pull/34462#discussion_r1420856548

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Make a PR adding some PHP 8.2 syntax in a file in `tools/` and `.github/`, and see if CI is still happy.
